doc/use_cli_in_airgapped_clouds.md

```diff
@@ -24,7 +24,8 @@ MSI | https://azurecliprod.blob.core.windows.net/msi/azure-cli-2.x.x.msi | Start
 **Note**: The CLI RPM package depends on a `python3` package and you'll need to install it separately while DEB and MSI packages already have a bundled Python in them.
 
 If you need to install and use Azure CLI in your pipeline, you could upload the Azure CLI package in a storage account that is accessible in the airgapped cloud, then you can download the package from the storage account and install it in your pipeline scripts. For instance, an Azure CLI deb package can be downloaded and installed with the following command:
-```
+
+```console
```

Comment:
More specifically, this is actually a bash command as `dpkg` only works in bash.

Reply:
Hi @jiasli 

I've made the requested change.